### ISSUE TITLE
R0/RP1 Configs

### ISSUE THREAD

**Issue by Uncreative426** [damonvv/Kerbal-Reusability-Expansion#4]
I created some RP1/R0 configs for KRE and was told to share them here and bring it to the Devs attention incase they wanted to ingrate it with the rest of the mod

https://www.dropbox.com/scl/fi/y0ay20g14gc67c5pwf7nd/KRE.cfg?rlkey=3ghtxyzu3b5w0j184vf6y9nv5&dl=0

^The link to the configs^ 

The only issue is that to get the gridfins to show up properly you have to go to your Game data/KerbalReusabilityExpansion folder.

then you need to open the GridFins folder and inside there should be 3 .cfg files open them up in notepad and at the top where it says the name put dashes between each of the words so it looks something like this

"Grid-Fin-L"

then repeat the steps in the GridFinsTitanium folder so they look something like this

"Grid-Fin-L-Titanium"

**Comment by TundraMods** [comment]
Hey! I added the patch. I've fixed the naming issue with adding a "?" in the patch config instead of editing the part configs. This will make sure the parts are not messing with the savefiles. Otherwise they will break crafts. You can take a look here:
https://github.com/TundraMods/Kerbal-Reusability-Expansion/commit/09643c6ae0b57a02fd36829b54463de346173d91

(You need the original install of KRE for the fins to work now instead of your edited configs. If it works, I'll add it to the release!

**Comment by Uncreative426** [comment]
Hey! So I looked at the config and there were issues with the grid fins showing up in the tech tree but I was able to fix that by adding ? at the end in the RP1 section like you did in the R0 section

https://www.dropbox.com/scl/fi/qiy19epwlrs5jpvd8o5rr/KRE_config.cfg?rlkey=7aq18fihgirz22a5ksoffjar8&st=xzr57nie&dl=0

I updated the configs though and now everything is working with a fresh install of KRE
also thank you for making such an amazing mod

**Comment by SofieBrink** [comment]
Would it be possible for you to simply make the required changes right here on github instead and submit a pull request for them? making changes and sending the file through dropbox, especially such a small file is quite unconventional.

**Comment by Uncreative426** [comment]
Ive never used github so Its kinda new to me

**Comment by SkyPhoenix999** [comment]
Hey I've already made these configs before

But if you want to make them official then you're going to want to resize the gridfins and legs since those parts have different rescale factors while reusing the same model, having everything with a rescale factor of 1.6 will just make them all the same size.
https://github.com/SkyPhoenix999/TundraExploration_RO

**Comment by Uncreative426** [comment]
Yep. I did notice that and Its already fixed in the last link I sent but thank you

**Comment by TundraMods** [comment]
This would do I think?
https://github.com/TundraMods/Kerbal-Reusability-Expansion/commit/1a8ed70732114983edb5773dffb433c6cc375f0d

**Comment by Uncreative426** [comment]
I just read through it but yea it looks like that has all the changes and would work
